test/helpers/user.ts

```diff
@@ -90,6 +92,17 @@ export const createUserWallet = async (index: number) => {
         }
       : null
     userRepo = await createUser({ phone, phoneMetadata })
+    if (userRepo instanceof Error) throw userRepo
+
+    if (entry.additionalWallets) {
+      for (const { currency } of entry.additionalWallets) {
```

Comment:
`currency` is type `any` here coming out of `entry.additionalWallets`. Could potentially be a good candidate for a `checkedToCurrency` validator.

Reply:
that is true but it's for the tests only, and I think the best way to manage this is to have a yaml validator, which will be a big PR by itself, so I guess to keep it as is for now.